components/Bounty/api/fetchOpenBounties.ts

```diff
@@ -0,0 +1,55 @@
+import API from "~/config/api";
+import { buildApiUri } from "~/config/utils/buildApiUri";
+import { Helpers } from "@quantfive/js-web-config";
+import { ID, User } from "~/config/types/root_types";
+
+type Args = {
+  onError: (error: Error) => void;
+  onSuccess: (response: any) => void;
+  page?: number;
+};
+
+/* BE may return more than specified below */
+export type SimpleBounty = {
+  amount: string;
+  content_type: any /* intentional any */;
+  created_by: User;
+  expiration_date: string;
+  id: ID;
+  item: any /* intentional any */;
+};
+
+export const fetchOpenBounties = ({ onError, onSuccess, page }: Args): void => {
```

Comment:
A couple of things. 

1. We already have a function called `fetchBountiesAPI` which does just that. I think you can tweak it to return a particular status
2. We should use the Bounty class as a type instead of something like the type defined above. If it doesn't do what you need, you can tweak it. You should be able to do `bounties.map(b => new Bounty(b))`
3. The Bounty class currently holds API logic + data. I think we should fetch through a method defined on the class to simplify things on the FE. I think it would be nice to do something like `Bounty.fetchOpen()` and have the method return a list of Bounty objects. That object already has three API endpoints it hits